tests/suites/os/tests/fsck/index.js

```diff
@@ -44,6 +45,20 @@ module.exports = {
 						);
 				}
 
+				async function checkTune2fsVersion(context){
```

Comment:
this could be extracted to be defined in the `suite.js` maybe, as a `checkUtilVersion(util, version){}` perhaps, if we see the need for it again